MainRepository/Program.cs

```diff
@@ -1,7 +1,21 @@
-﻿internal class Program
+﻿//SWAP Indices

```

Comment:
Alignment is completely wrong. Use "Ctrl + K" & "Ctrl + F" to view the difference.
And I'm unable to compile.